src/main/java/net/sf/jabref/gui/event/Event.java

```diff
@@ -0,0 +1,21 @@
+package net.sf.jabref.gui.event;
+
+/**
+ * Represents a medium which can be used
+ * to pass objects on event occurrence.
+ */
+
+public class Event {
```

Comment:
I see no reason for using java-util. We are implementing something contrary when using Guava.